include/RAJA/policy/cuda/params/kernel_name.hpp

```diff
@@ -19,7 +19,7 @@ template<typename EXEC_POL>
 camp::concepts::enable_if<RAJA::type_traits::is_cuda_policy<EXEC_POL>>
 param_init(EXEC_POL const&, KernelName& kn, const RAJA::cuda::detail::cudaInfo&)
 {
-#if defined(RAJA_ENABLE_NV_TOOLS_EXT)
+#if defined(RAJA_ENABLE_NV_TOOLS_EXT) && !defined(RAJA_ENABLE_CALIPER)
   nvtxRangePush(kn.name);
```

Comment:
This seems like it could cause confusion as most people will probably not be aware of the consequences of mixing these options.

Reply:
@LLNL/raja-core , folks when caliper is not enabled should we still enable vendor specific ranges?